client/my-sites/email/email-non-domain-owner-message/index.tsx

```diff
@@ -96,7 +96,7 @@ export const EmailNonDomainOwnerMessage = ( props: EmailNonDomainOwnerMessagePro
 		},
 		args: {
 			ownerUserName,
-			selectedDomainName: domain?.name,
+			selectedDomainName: domain?.name ?? '',
```

Comment:
Does it make sense if we default this to `'this domain'`? cc @Automattic/nomado for opinion

Reply:
Looking through the consuming code, I'm fairly sure a domain will always exist -- I'd recommending refining the types to reflect that if true!